src/main/java/core/basesyntax/UserService.java

```diff
@@ -2,10 +2,16 @@
 
 public class UserService {
     public void registerUser(User user) {
-
+        PasswordValidator passwordValidator = new PasswordValidator();
```

Comment:
FYI: 
it's better to declare it as class field and not local variable, cause it won't create new object every time we call this method